pom.xml

```diff
@@ -149,6 +149,33 @@
 	</profiles>
 
 	<build>
+		<resources>
+		    <resource>
+		        <filtering>false</filtering>
+		        <directory>src/main/resources</directory>
+		        <excludes>
+		            <exclude>**/web-fragment.xml</exclude>
+		        </excludes>
+		    </resource>
+		    <resource>
+		        <filtering>true</filtering>
+		        <directory>src/main/resources</directory>
+		        <includes>
+		            <include>**/web-fragment.xml</include>
+		        </includes>
+		    </resource>
+		    <resource>
+		        <directory>${maven.multiModuleProjectDirectory}</directory>
+		        <includes>
+		            <include>RELEASE_NOTES</include>
+		        </includes>
+		        <targetPath>${project.build.outputDirectory}/META-INF</targetPath>
+		    </resource>
+		    <resource>
+		        <targetPath>META-INF/resources</targetPath>
+		        <directory>src/main/webapp</directory>
```

Comment:
@joaojtmarques-qubit non blocking for accepting the PR but can we switch the order between **directory** and **targetPath** since directory is the source and target path is the target.